go.mod

```diff
@@ -96,6 +97,6 @@ replace (
 	github.com/earthly/earthly/ast => ./ast
 	github.com/jessevdk/go-flags => github.com/alexcb/go-flags v0.0.0-20210722203016-f11d7ecb5ee5
 
-	github.com/moby/buildkit => github.com/earthly/buildkit v0.0.1-0.20220913002628-9c256c19c539
-	github.com/tonistiigi/fsutil => github.com/earthly/fsutil v0.0.0-20220719234708-392da7eebeb5
+	github.com/moby/buildkit => github.com/earthly/buildkit v0.0.1-0.20221026205052-75bffa4fcb41
+	github.com/tonistiigi/fsutil => github.com/earthly/fsutil v0.0.0-20221019181234-420dfa996232
```

Comment:
this seems off; fsutils is at `b994beadd44391cd559120cd153f2b7de68d14c6`

Reply:
Good catch!
I must have forgotten to update the SHA to the `earthly-main` merge commit.
This has been fixed now.